<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Syful-Lenovo" failures="1" tests="5" name="Car Loan" time="0.074" errors="0" timestamp="2018-11-17T20:56:33 EST">
  <testcase classname="testNGtutorial.day3" name="weblogin" time="0.002"/>
  <testcase classname="testNGtutorial.day3" name="weblogin" time="0.0"/>
  <testcase classname="testNGtutorial.day3" name="weblogin" time="0.001"/>
  <testcase classname="testNGtutorial.day3" name="mobilesignincanlogin" time="0.005">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at testNGtutorial.day3.mobilesignincanlogin(day3.java:44)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- mobilesignincanlogin -->
  <testcase classname="testNGtutorial.day3" name="mobilesignoutcanlogin" time="0.0"/>
  <testcase classname="testNGtutorial.day3" name="mobilelogincarlogin" time="0.0">
    <ignored/>
  </testcase> <!-- mobilelogincarlogin -->
  <testcase classname="testNGtutorial.day3" name="APIcarLoan" time="0.0">
    <ignored/>
  </testcase> <!-- APIcarLoan -->
</testsuite> <!-- Car Loan -->
